### ISSUE TITLE
Bulk Reassignment Functionality All Organizations in Queue

### ISSUE THREAD

**Issue by araposo-tistatech** [department-of-veterans-affairs/caseflow#15090]
## User or job story
User story: As an admin user of an organization, I should have the ability to bulk reassign tasks to myself or other users part of my organization, so that I can quickly and efficiently reassign tasks that need to be moved. 

## Acceptance criteria
- [ ] Enable initially for just: Ann-Marie to provide
- [ ] Roll out to all orgs after testing
- [ ] Only admin users of organizations can see bulk reassign button on the assigned tab
- [ ] Only admin users are able to bulk reassign tasks to themselves or another team member
- [ ] Confirmation Banner displays on the Team cases page once the task reassignment action is complete
- [ ] If task reassignment action cannot be completed an alert message or banner displays stating the tasks could not be reassigned with reason

## Release notes

Admin users of the following organizations can now bulk reassign tasks to themselves or their team members.

AOD
VSOs
Board Dispatch
Board of Veterans' Appeals
Case Review
Hearing Admin
Hearings Management
Litigation Support
Mail
Privacy Team
Quality Review
Transcription
Translation
VLJ Support Staff

### Background/context

The BVA Dispatch team has identified a functionality gap in Caseflow. When a team member is on leave they need to be able to move all of that team members tasks to another team member so the tasks can be worked and re not delayed. As they will more often then not need to move more than one task they are requesting a feature to be able to bulk reassign tasks (similar to bulk assign). 

### Technical notes

Organizations this change should apply to: 
AOD
American Legion
American Red Cross
American Veterans
Armed Forces Services Corporation
Blinded Veterans Association
Board Dispatch
Board of Veterans' Appeals
Case Review
Disabled American Veterans
Fleet Reserve Association
Hearing Admin
Hearings Management
Litigation Support
Mail
Marine Corps League
Maryland Department of Veterans Affairs
Military Order of the Purple Heart
Paralyzed Veterans of America
Privacy Team
Quality Review
Transcription
Translation
VLJ Support Staff
Veterans of Foreign Wars
Vietnam Veterans of America
Virginia Department of Veterans Services
Wounded Warrior Project 

### Other notes
Inquiring w/the Board if teams would ever reassign cases that are on hold. If so, what does that mean for Caseflow?
